ntcore/src/main/java/edu/wpi/first/networktables/NetworkTable.java

```diff
@@ -325,15 +404,6 @@ public Set<String> getSubTables() {
     return keys;
   }
 
-  /**
-   * Deletes the specified key in this table. The key can not be null.
-   *
-   * @param key the key name
-   */
-  public void delete(String key) {
```

Comment:
Reduced functionality?

Reply:
Yep, it's gone.  It's not possible to delete topics.